packages/framework/presence/src/test/presenceManager.spec.ts

```diff
@@ -113,114 +104,216 @@ describe("Presence", () => {
 					});
 				});
 
-				it("is announced via `attendeeJoined` when new", () => {
-					// Act - simulate join message from client
-					presence.processSignal("", initialAttendeeSignal, false);
-
-					// Verify
-					assert(newAttendee !== undefined, "No attendee was announced");
-					assert.equal(
-						newAttendee.sessionId,
-						newAttendeeSessionId,
-						"Attendee has wrong session id",
-					);
-					assert.equal(
-						newAttendee.getConnectionId(),
-						initialAttendeeConnectionId,
-						"Attendee has wrong client connection id",
-					);
+				it("is not announced via `attendeeDisconnected` when unknown connection is removed", () => {
+					// Setup
+					presence.events.on("attendeeDisconnected", () => {
+						assert.fail("No attendee should be disconnected in beforeEach");
+					});
+
+					// Act & Verify - remove unknown connection id
+					presence.removeClientConnectionId("unknownConnectionId");
 				});
 
-				describe("disconnects", () => {
-					let disconnectedAttendee: ISessionClient | undefined;
+				describe("is joining", () => {
+					let newAttendee: ISessionClient | undefined;
 					beforeEach(() => {
```

Comment:
if there is only one case, then don't need beforeEach. Will there be more cases to test?